workflow/automation/estimation/estimate_cybershake.py

```diff
@@ -457,34 +351,148 @@ def main(args):
         (r_fd_counts[:, None], r_hf_nt[:, None], r_bb_ncores[:, None]), axis=1
     )
 
+    print("Preparing IM_calc input data")
+    if root_config["ims"][const.RootParams.extended_period.name]:
+        period_count = len(
+            np.unique(np.append(root_config["ims"]["pSA_periods"], const.EXT_PERIOD))
+        )
+    else:
+        period_count = len(root_config["ims"]["pSA_periods"])
+    im_calc_input_data = [
+        len(get_stations(root_config["stat_file"])),
+        np.repeat(fault_sim_durations / float(root_config["dt"]), r_counts),
+        root_config["ims"][const.SlBodyOptConsts.component.value],
+        period_count,
+        platform_config[const.PLATFORM_CONFIG.IM_CALC_DEFAULT_N_CORES.name],
+    ]
+
     results_df = run_estimations(
         fault_names,
         realisations,
         r_counts,
-        model_dir_dict,
         lf_input_data,
         hf_input_data,
         bb_input_data,
+        im_calc_input_data,
     )
 
     results_df["fault_name"] = np.repeat(fault_names, r_counts)
     return results_df
 
 
-if __name__ == "__main__":
-    parser = ArgumentParser()
+def display_results(df: pd.DataFrame, verbose: bool = False):
+    """Displays the results in a nice format.
+    If verbose is specified the results are shown on a per fault basis.
+    """
+    print()
+    process_types = "{:>14}{:>30}{:>30}{:>35}".format("LF", "HF", "BB", "IM_calc")
+    if verbose:
+        header = "{:<12}{:<10}{:<8}".format("core hours", "run time", "cores")
+        print(process_types)
+        print("{:>12}{}{}{}{}".format("", header, header, header, header))
+        process_type_result = "{:<12.3f}{:<10.3f}{:<8.0f}"
+        for fault_name, row in df.groupby("fault_name").sum().iterrows():
+            cols = (
+                const.MetadataField.core_hours.value,
+                const.MetadataField.run_time.value,
+                const.MetadataField.n_cores.value,
+            )
+            lf_str = process_type_result.format(
+                *[row.loc[const.ProcessType.EMOD3D.str_value, col] for col in cols],
```

Comment:
It shouldn't need the list comprehension, just
*row.loc[const.ProcessType.EMOD3D.str_value, cols].values

Reply:
I found when I ran this it didn't work, just did some further debugging, realized I needed to have cols in [] not ()
![image](https://user-images.githubusercontent.com/33056240/173969391-7dd9340b-6708-4bf5-883e-fc2af9e9cffa.png)

Reply:
Huh, that's wild, I would have expected list and tuple to have the same behaviour ther